sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/ApproximateUnique.java

```diff
@@ -190,6 +200,15 @@ public Globally(double maximumEstimationError) {
             + "estimation error between 1% (0.01) and 50% (0.5).");
       }
       this.sampleSize = sampleSizeFromEstimationError(maximumEstimationError);
+
+      final double finalMaxEstError = maximumEstimationError;
+      this.displayDataCallback = new HasDisplayData() {
+
+        @Override
+        public void populateDisplayData(DisplayData.Builder builder) {
+          builder.add("maximumEstimationError", finalMaxEstError);
```

Comment:
I'm not sure I'm a fan of the `displayDataCallback` approach. I think it might be clearer to put the maximumEstimationError in the class (as a `@Nullable` field) and then have the populateDisplayData attach that if it is non-null. Also, it seems useful to include the computed `sampleSize`.

It may also be useful to have versions of `add` that omit null values?